### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by Anna-Myzukina**

### CODE REVIEW

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey   @Anna-Myzukina @maya88en   Excellent Job so far on your final Html project the site looks lovely :100: 
image. 

There are just some changes to  checkout below.

##### Think of this project more like bootstrap you have a class for most things example `text-primary`

### Box utillties 
- [ ] The CSS files define separate classes for margin top, bottom, left, right, all margins and auto

 - [ ]  The margins should span from 0 rem to 1 rem
 - [ ] The margins should have breakpoints (e.g. margin-sm-top-1)


#### Padding


 - [ ] The CSS files define separate classes for padding top, bottom, left, right, all padding and auto

 - [ ] The padding should span from 0 rem to 1 rem


- [ ] The padding should have breakpoints (example padding-lg-2)


### Positioning

#### Floats

* [ ] The CSS files define separate classes for float left, right and clearing the floats

#### Positions

* [ ] You should define separate classes to change the position property of an element to relative, absolute or fixed

### Display

#### Display properties

The CSS files define:

* [ ] A class to set the display property of an element to flex
* [ ] A class to set the display property to hidden
* [ ] A class to set the display property to none
* [ ] The none class should have breakpoints (example display-md-none)

#### Flex properties

The CSS files define:

* [ ] Classes to set vertical alignment as top, bottom or center
* [ ] Classes to set horizontal alignment as left, right or center

### Colors

#### Background and text

The CSS files define:

* [ ] At least ten different classes to set the background and text colors of an element

### Text

#### Utilities

The CSS files define:

* [ ] Classes to horizontally align text to the left, right or center
* [ ] At least five classes to set the font size of an element
* [ ] Three classes to set the font weight of an element (bold, normal, medium)

#### Border

* [ ] The CSS files define separate classes for border top, bottom, left, right, all border
* [ ] The border should span from 0 to 1rem
* [ ] The border should have classes for making box round
* [ ] The border should have a class defined to transform a box into a circle

### Images 
- [ ] your images are not showing up on the site. The folder **smashing magazine** seems to be missing.

#html errors 
 There are some of HTML errors on this page use this tool and please correct them https://validator.w3.org/

_Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description._

**Review by stratospherique — CHANGES_REQUESTED** [review]
#### Splendid work so far :clap: :clap: . Unfortunately, some changes that prevents your project's approval needs to be implemented: 

#### Margins

* [ ] The margins should have breakpoints (e.g. margin-sm-top-1)

#### Padding

* [ ] The padding should have breakpoints (example padding-lg-2)

- for padding and margin you need to add media queries.

### Display
#### Display properties
The CSS files define:

* [ ]  A class to set the display property of an element to flex
* [ ]  The none class should have breakpoints (example display-md-none)

### Text
#### Utilities
The CSS files define:

* [ ]  At least five classes to set the font size of an element
* [ ]  Three classes to set the font weight of an element (bold, normal, medium)

### Border
* [ ]  The border should span from 0 to 1rem
* [ ]  The border should have classes for making box round
* [ ] The border should have a class defined to transform a box into a circle

### Images
 * [ ] your images are not showing up on the site. The folder smashing magazine seems to be missing.

### html errors
There are some of HTML errors on this page use this tool and please correct them https://validator.w3.org/


## And that's it. Your project is amazing and implementing the above would make it extraordinary :smile: . When finished, kindly resubmit another code review request. Keep UP the good work :muscle: :muscle: .

File: index.html [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+                        id adipisci molestiae ipsam ea laborum l
+
+                    </p>
+                </div>
+            </article>
+            <div class="row person">
+                <article class="col-large-6 col-medium-12">
+                    <div class="row">
+                        <div class="col-medium-2 persons_photo">
+                            <img src="../Smashing-Magazine/assets/img/rachel-andrew-tw-profile.jpg" alt="Persons photo">
```
Comment (samgaco) [inline index.html:416]: You still have to correct the route to the images so it shows when someone clones your project.

**Review by samgaco — APPROVED** [review]
### Hi Good job implementing the requested changes. Your project is approved :+1: Congratulations :tada: 

Please :pray:  before merging, it would be suggested that you address the issue with the images.

**Pull request merged**